cli/src/config.ts

```diff
@@ -193,7 +194,17 @@ export class Config implements CliConfig {
   }
 
   private initWindowsConfig() {
-    this.windows.androidStudioPath = this.app.windowsAndroidStudioPath && this.app.windowsAndroidStudioPath;
+    if (this.app.windowsAndroidStudioPath) {
+      if (!existsSync(this.app.windowsAndroidStudioPath) && this.cli.os === OS.Windows) {
+        const buffer = execSync('REG QUERY "HKEY_LOCAL_MACHINE\\SOFTWARE\\Android Studio" /v Path');
```

Comment:
can you put all this in a try catch?
If Android Studio is not installed it shows an ugly message because it doesn't find the registry.
In the catch you can set the path to empty string so it can be used later in the open command to not open if it's empty. I'll do that in a separate PR once this is merged.